_includes/sections/working-here/redundancy-policy.md

```diff
@@ -0,0 +1,134 @@
+### Redundancy Policy
+
+The coronavirus pandemic has caused us to look at some of our HR policies - or
+more pertinently, the lack of them. It comes at a time where we have grown as a
+company, but have not yet put in place all the policies and procedures that a
+larger organisation might need.
+
+This policy is being put in place now, whilst we’re not facing any imminent need
+to make redundancies - in a classic "fix the roof whilst the sun is shining"
+way.
+
+In putting it together, we have considered not just our legal obligations, but
+how best to go about doing a hard thing, in line with our dxw values.
+
+#### Policy statement
+
+We believe that continuity of employment and job security is important for staff
+wellbeing, and for delivering quality work to our clients. We work hard to keep
+our sales pipeline healthy. And we plan the future development of dxw carefully
+so we can hire the right people at the right time.
+
+However, the future is never certain. Significant changes in our plans or in the
+markets we serve, could mean that we need to make a role redundant. If that
+happens, we will follow a fair and open process.
+
+This section of the playbook sets out what we’ll do in those circumstances.
+
+#### How roles are selected for redundancy
+
+We will take an open, data-led approach in redundancy situations. We might, for
+example, find that we need to reduce our unbillable roles to keep profits at a
+sustainable level. Or that we need less people in a particular specialism if we
+experience a reduction in demand for that type of work.
+
+If a group of people in very similar roles are involved in the redundancy
+situation, everyone in the pool will be treated as being at risk and there will
+be a selection process. We will publish the selection criteria at the time,
+which may include any or all of the following:
+
+- A performance assessment
+- Any disciplinary issues having previously arisen
```

Comment:
"Any past disciplinary issues" ?

Reply:
@Possapostolou has asked to sign off the text as it is for now, hope that's ok